first e2e test in `TestE2E/BUILD/From/` fails with `privileges escalation failed: operation not permitted`

## Version of Singularity:

master

This one took a bit to find and I"m not certain what the issue is.

If you run the e2e tests, the first test case always fails due to the privilege increase. The rest of the tests work as expected.

```
    --- FAIL: TestE2E/BUILD (186.42s)
        imgbuild.go:652: {/usr/local/singularity/bin/singularity /tmp/stest.673844731}
        --- FAIL: TestE2E/BUILD/From (55.45s)
            --- FAIL: TestE2E/BUILD/From/DockerURIDup (0.00s)
                imgbuild.go:50: privileges escalation failed: operation not permitted

     --- PASS: TestE2E/BUILD/From/BusyBox (1.98s)
     ...
     --- SKIP: TestE2E/BUILD/From/Debootstrap (0.00s)
     imgbuild.go:53: debootstrap not found in path
     --- PASS: TestE2E/BUILD/From/DockerURI (3.26s)
     ...
```

Notes:
- I added `DockerURIDup` as a duplicate case before the `BusyBox` test case to verify the 1st one always fails. Previously `BusyBox` would always fail. 
- Note that the later `DockerURI` case passes where as the first one fails.

This test is defined in `e2e/imgbuild/imgbuild.go`, `buildFrom`, but it is failing in the `e2e.Privileged` function where it tries to set `uid/gid`.

This is currently causing CI failures when pull requests are merged in.

If you re-run the tests with `-v` set for verbose on the `go test` line it should be more obvious what is and is not failing.

--- a/e2e/inspect/inspect.go
+++ b/e2e/inspect/inspect.go
@@ -11,7 +11,6 @@ import (
 
 	"github.com/buger/jsonparser"
 	"github.com/sylabs/singularity/e2e/internal/e2e"
-	"github.com/sylabs/singularity/internal/pkg/test"
 )
 
 type testingEnv struct {
@@ -118,7 +117,7 @@ func singularityInspect(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		t.Run(tt.name, test.WithoutPrivilege(func(t *testing.T) {
+		t.Run(tt.name, func(t *testing.T) {
 			// Inspect the container, and get the output
 			out, err := runInspectCommand(tt.insType)
 			if err != nil {
@@ -135,7 +134,7 @@ func singularityInspect(t *testing.T) {
 				t.Fatalf("unexpected failure: got: %s, expecting: %s", v, tt.expectOut)
 			}
 
-		}))
+		})
 	}
 
 }

